src/scripts/ci/publish.ts

```diff
@@ -294,33 +294,37 @@ export function getPublishOrder(packages: Packages): string[][] {
 }
 
 /**
- * Takes the max alpha version + 1
- * For now supporting 2.0.0-alpha.X
+ * Takes the max dev version + 1
+ * For now supporting 2.0.0-dev.X
```

Comment:
```suggestion
 * For now supporting 2.Y.Z-dev.#
```
"naming" via https://www.notion.so/prismaio/Prisma2-Release-Publishing-Process-Jan-2020-04-a9264481ba934ed79cab4e35934523a4#2d5ddd0d1402466c8af67457cd2b9841